test/quantum/run_strategy_test.exs

```diff
@@ -0,0 +1,31 @@
+defmodule Quantum.RunStrategyTest do
+  use ExUnit.Case, async: true
+
+  alias Quantum.Job
+  alias Quantum.RunStrategy.NodeList
+
+  defmodule Scheduler do
+    @moduledoc false
+
+    use Quantum.Scheduler, otp_app: :quantum_test
+  end
+
+  test "run strategy local" do
+    job = Job.new(Scheduler.config(run_strategy: Quantum.RunStrategy.Local))
+    assert %Job{} = job
+    assert [_] = NodeList.nodes(job.run_strategy, job)
+  end
+
+  test "run strategy random" do
+    node_list = [:node1, :node2]
+    job = Job.new(Scheduler.config(run_strategy: {Quantum.RunStrategy.Random, node_list}))
+    assert [node] = NodeList.nodes(job.run_strategy, job)
+    assert Enum.member?([:node1, :node2], node)
+  end
+
+  test "run strategy all" do
+    node_list = [:node1, :node2]
+    job = Job.new(Scheduler.config(run_strategy: {Quantum.RunStrategy.All, node_list}))
+    assert [_, _] = NodeList.nodes(job.run_strategy, job)
```

Comment:
Could you assert here that both `:node1` and `:node2` are present in the returned array?